app/models.py

```diff
@@ -0,0 +1,8 @@
+from dataclasses import dataclass
+
+
+@dataclass
+class Actor:
+    id: id
```

Comment:
The type annotation for `id` is incorrect. It should be `int` instead of `id`. This will ensure that the `id` field is expected to be an integer, which is typical for database IDs.